initrd/bin/gui-init

```diff
@@ -374,6 +375,12 @@ while true; do
     continue
   fi
 
+  if [ "$totp_confirm" = "S" ]; then
+    whiptail --title 'Systeminfo' \
+      --msgbox "$(cat /proc/meminfo | grep 'MemTotal' | sed 's/MemTotal:/Mem:/')\n\n$(cat /proc/cpuinfo | grep 'model name' | uniq | sed 's/model name.*:/CPU:/')\n\n$(fdisk -l | grep '/dev/sd.:' | sed 's/B,.*/B/')" 16 60
```

Comment:
would be useful to display the Heads/firmware version as well.
also, should include NVMe disks in the fdisk call